Cargo.toml

```diff
@@ -19,6 +19,7 @@ hashbrown = "0.15"
 hashlink = "0.10"
 indexmap = "2"
 intrusive-collections = "0.9.7"
+ordermap = "1"
```

Comment:
This should be a feature gated dependency at the very least

Reply:
https://github.com/salsa-rs/salsa/pull/1034